### ISSUE TITLE
Visualization from Token Counts or Frequencies

### ISSUE THREAD

**Issue by morden96** [JasonKessler/scattertext#23]
Recommendation:
MapReduce is typically used to return just word counts in large datasets. While much more simplistic than what scattertext provides, it is very fast on large datasets. However, your method of visualizing the data is stellar. It would be ideal if we could leverage scattertext_explorer using word count or frequency tables.

**Comment by gabefair** [comment]
Are you suggesting an ability to use a dataset that already has the word frequencies counted? Instead of using `get_term_freq_df` to get the word count?

**Comment by morden96** [comment]
Yes. Allow the user to create their own frequency or wordcounts and then leverage scattertext for the analysis/visualization. Otherwise, scattertext is unusable with large datasets, particularly ones that cannot fit in memory.

**Comment by gabefair** [comment]
This is exactly what I need!

**Comment by JasonKessler** [comment]
Thanks for the feedback.  Since it seems like there's a lot of demand for this, I'll put together something to make plots based on a term-category frequency dataframe.
